simtbx/kokkos/SConscript

```diff
@@ -127,6 +127,18 @@ if use_sycl:
   kokkos_env.Replace(CXXFLAGS=[list(OrderedDict.fromkeys(kokkos_env['CXXFLAGS']))])
   kokkos_env.Replace(SHLINKFLAGS=[list(OrderedDict.fromkeys(kokkos_env['SHLINKFLAGS']))])
 
+if True: # diffuse code in exascale API
```

Comment:
should this if test be removed ?

Reply:
@dermen Incoming commit explains in a comment that the block is indented to highlight code that can be removed in the future if we ever finish replacing the eigen dependency with the newer kokkoslib code.